modules/gis/src/lib/geojson-to-binary.ts

```diff
@@ -25,9 +25,12 @@ export const TEST_EXPORTS = {
   secondPass
 };
 
+type NumericArrayConstructor = Float32ArrayConstructor | Float64ArrayConstructor | ArrayConstructor;
```

Comment:
Nit: Define next to `NumericArray` type and cover all cases? i.e. any reason not to handle integer typed arrays>

Reply:
You mean into `modules/loader-utils/src/types.ts` and then import it? That doesn't seem to follow what the rest of the modules do, for example there are many places where `TypedArray` is defined only for that module

Reply:
Yes typing this stuff is a never ending effort... I was trying to consolidate such things in the `loaders.gl/schema` module but this can come later.